test/unit/knex.js

```diff
@@ -447,6 +447,13 @@ describe('knex', () => {
       return knex.destroy();
     });
 
+    it('supports transaction isolation', async () => {
```

Comment:
Would be better to run it in an integration test to cover all the dbs. See test/integration2 for a reference.
If you have Docker installed, you can use `npm run db:start:no-oracle` to have all the dbs locally.
Also test needs to ensure that isolation level is actually doing something.